openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+root="$(dirname "${BASH_SOURCE[0]}")/../.."
+
+# Source the main vars file to get the serving/eventing version to be used.
+source "$root/hack/lib/vars.bash"
+
+serving_files=(serving-crds serving-core serving-hpa serving-post-install-jobs)
+eventing_files=(eventing-crds eventing-core in-memory-channel mt-channel-broker eventing-sugar-controller eventing-post-install-jobs)
```

Comment:
should we add a comment, that these could, in theory change?  the list of "related" file for the job?

I guess one more "moving target" for the "release doc" from @rhuss

Reply:
Done.

Reply:
should we invoke that script via the `Makefile` :thinking:

Reply:
ultimately, yes. I'd like to connect all scripts after @cardil's #490 gets in and/or it should be called from the target that he's creating imo. The kourier script is another candidate.